distributed-databases/trino/providers.tf

```diff
@@ -0,0 +1,32 @@
+provider "aws" {
```

Comment:
Move contents from the `providers.tf` file into `main.tf`, and after that, you can delete the `providers.tf` file.

Reply:
addressed